server/controllers/users/lib/users.js

```diff
@@ -0,0 +1,24 @@
+const CONFIG = require('config')
+const __ = CONFIG.universalPath
+const _ = __.require('lib', 'utils')
+const error_ = __.require('lib', 'error')
+const promises_ = __.require('lib', 'promises')
+
+module.exports = {
+  validateData: function (data) {
+  },
+  findOrCreateUser: function(userInfo, done){
+    // mock user api (see branch 416-user-api)
```

Comment:
Let's integrate `416-user-api` into `master` and rebase this PR against it then before merging.